docs/readme_chunks/generate_frontpage_documents.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
I feel like I am missing some context here - why is the reading/chunking  needed to generate the docs?